front/components/assistant/conversation/MessageItem.tsx

```diff
@@ -112,46 +118,101 @@ const MessageItem = React.forwardRef<HTMLDivElement, MessageItemProps>(
       case "user_message":
         const citations = message.contenFragments
           ? message.contenFragments.map((contentFragment) => {
-              const citationType = [
-                "dust-application/slack",
-                "text/vnd.dust.attachment.slack.thread",
-              ].includes(contentFragment.contentType)
-                ? "slack"
-                : "document";
+              let visual;
```

Comment:
Note: this is temp, will move and centralize this logic into one common component with the InputBarAttachments